lib/ox/element.rb

```diff
@@ -150,7 +150,18 @@ def locate(path)
     # _raise_ [NoMethodError] if no match is found
     def method_missing(id, *args, &block)
       has_some = false
-      ids = id.to_s
+      ids = id.to_s.sub('_', ':') # allows easy API with namespaces
+      
+      if ids[-1] == '!' # return array of elements matching ids
```

Comment:
I'd prefer that too, but I meant for `doc.somewhere.else.each{|n| ... }` to yield all the `else` elements on the `somewhere` node. But `doc.somewhere.else` will have evaluated to the first `else` node on `somewhere` node, and from that `else` node there's no way to find all its sibling `else`s, since from that node you can't get to the parent to find those siblings. That's why I used `else!`; it doesn't evaluate to a node so I stay in the context of `somewhere` and I can access its children there.